.github/scripts/version_updater.sh

```diff
@@ -67,7 +68,16 @@ if [[ -z "${role}" ]]; then
 fi
 
 # Get latest version.
-version="$(github_api "repos/${source_repo}/releases/latest" | jq '.tag_name' | tr -d '"v')"
+if [[ "${type}" == "github" ]]
+then
+  version="$(github_api "repos/${source_repo}/releases/latest" | jq '.tag_name' | tr -d '"v')"
+elif [[ "${type}" == "gitlab" ]]
+then
+  version="$(curl --retry 5 --silent --fail "https://gitlab.com/api/v4/projects/${source_repo}/releases"|jq '.[0].tag_name'| tr -d '"v')"
```

Comment:
Perhaps move the gitlab api url to a function named gitlab_api() to have it in line with the existing github_api() function.

Reply:
I thought about it. I thought this would be an overcomplication of the code.
Okay, I pushed update )